Client/Frontend/Fakespot/Views/FakespotHighlightsCardView.swift

```diff
@@ -141,6 +150,36 @@ class FakespotHighlightsCardView: UIView, ThemeApplicable {
         dividerView.backgroundColor = theme.colors.borderPrimary
     }
 
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        guard let viewModel else { return }
+        guard viewModel.isOneHighlightGroupWithTwoReviews else { return }
+        guard let longestReview = viewModel.longestTextFromReviews else { return }
+
+        let highlightLabelWidth = longestReview.width(
+            with: DefaultDynamicFontHelper.preferredFont(
+                withTextStyle: .subheadline,
+                size: UX.titleFontSize,
+                weight: .semibold
+            )
+        )
+
+        let highlightsGroupViewWidth = (bounds.width - UIWindow.safeAreaInsets.left * 2) -
```

Comment:
Please add a comment on this calculation. Also we shouldn't force unwrap anything here - you could add `highlightGroups.first` to the guards above. The magic number `24` should be replaced with a UX constant.